### PR TITLE
Fixed Facility Cover Image upload pop-up

### LINKED ISSUE #6610: Facility Cover Image upload pop-up is malfunctioning
**Describe the bug**

When we click anywhere on the places where i mentioned in the screenshot, the cover image pop-up is been opening up

**To Reproduce**

1. Go to facility detail page
2. Click on any specific place mentioned in the screen shot
3. See error

**Expected behavior**

The cover image pop-up should be only open when we click on the respective place

**Screenshots**

image

### PR DESCRIPTION
**Pull request by konavivekramakrishna**
### <samp>🤖 Generated by Copilot at 2700c58</samp>

Refactor edit cover image functionality for facilities. Remove onClick handler from `FacilityHome.tsx` and use a separate component and button instead.

## Proposed Changes

- Fixes #6610 
- Removed extra unnessary onClick action
@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### <samp>🤖 Generated by Copilot at 2700c58</samp>

* Remove the onClick handler for the facility cover image ([link](https://github.com/coronasafe/care_fe/pull/6616/files?diff=unified&w=0#diff-ad6af8429c47f2623c154aeb1256b603ed30a7b233d485bef095d3ddf7c4b97cL376-L378)) and replace it with a button that opens the edit cover image modal (

### CODE REVIEW

File: src/Components/Facility/FacilityHome.tsx [hunk 1]
```diff
@@ -373,9 +373,6 @@ export const FacilityHome = (props: any) => {
           className={`group relative h-48 w-full text-clip rounded-t bg-gray-200 opacity-100 transition-all duration-200 ease-in-out md:h-0 md:opacity-0 ${
             hasPermissionToEditCoverImage && "cursor-pointer"
           }`}
```
Comment (rithviknishad) [inline src/Components/Facility/FacilityHome.tsx]: You should also remove the `cursor-pointer`

```suggestion
          className="group relative h-48 w-full text-clip rounded-t bg-gray-200 opacity-100 transition-all duration-200 ease-in-out md:h-0 md:opacity-0"
```

**Comment by konavivekramakrishna** [comment]
@rithvik just a minor fix 
i will do it now

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Comment by rithdick** [comment]
Attention
This email is from an HWDSB Student.
________________________________
This is the wrong mail address

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**